src/services/reporting/carSales.ts

```diff
@@ -98,3 +98,21 @@ export const deleteCarSalesRejection = async ({
     })
   )
 }
+
+export const fetchCarSalesCount = async ({
+  dealerId,
+  status,
+  hasRejection,
+  options = {},
+}: {
+  dealerId: number
+  status?: string
+  hasRejection?: boolean
+  options?: ApiCallOptions
+}): Promise<CarSaleCount> => {
+  const query = toQueryString({ status, hasRejection })
+  return fetchPath({
+    path: `dealers/${dealerId}/car-sales/count${query ? `?${query}` : ""}`,
+    options: { isAuthorizedRequest: true, ...options },
+  })
+}
```

Comment:
How about returning a number directly?
```suggestion
}): Promise<number> => {
  const query = toQueryString({ status, hasRejection })
  const { count } =  fetchPath({
    path: `dealers/${dealerId}/car-sales/count${query ? `?${query}` : ""}`,
    options: { isAuthorizedRequest: true, ...options },
  })
  return count
}
```